src/engraving/libmscore/barline.cpp

```diff
@@ -1276,7 +1276,7 @@ void BarLine::layout()
     // after staff distances are known). This is a temporary layout.
     double _spatium = spatium();
     y1 = _spatium * .5 * _spanFrom;
-    if (RealIsEqual(y2, 0.0)) {
+    if (RealIsEqual(y2, 0.0) || score()->isPaletteScore()) {
```

Comment:
Could you please explain why we need this `score()->isPaletteScore()` here? I'm not a big fan of them. Too much knowledge of palettes in libmscore

Reply:
@RomanPudashkin I found an alternative solution that doesn't involve anything palette-specific. Probably this is a better solution anyway.